README.md

```diff
@@ -51,15 +51,18 @@ This script will configure your system to install global npm packages without ha
 
 ### Yeoman not found during project generation
 
-Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. We've found that this is sometimes caused by not having the node modules binaries route correctly configured.
+Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. This could be due to having more than one node version installed, or not having the right `PATH`. Either way, here is how to fix this issue.
```

Comment:
```suggestion
Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. This could be due to having more than one node version installed, or not having the right `PATH` configured. Either way, here is how to fix this issue.
```